packages/react-examples/src/react-tooltip/Tooltip/Tooltip.stories.tsx

```diff
@@ -1,58 +1,169 @@
+  },
+
   target: {
-    width: '128px',
-    height: '128px',
-    margin: '64px',
-    background: 'lightgray',
-    border: '1px solid darkgray',
+    width: '48px',
+    height: '48px',
   },
 });
 
-export const TooltipExample = () => (
-  <ThemeProvider theme={webLightTheme}>
-    <TooltipExampleCore />
-  </ThemeProvider>
-);
+export const Basic = () => {
+  const styles = useStyles();
+
+  const targetRef1 = React.useRef<HTMLDivElement>(null);
+
+  return (
+    <FluentProvider theme={webLightTheme}>
+      Hover or focus the buttons to show their tooltips:
+      <div className={styles.exampleList}>
+        <Tooltip content="This is a default tooltip" type="description">
+          <button>Default</button>
+        </Tooltip>
+        <Tooltip content="This is a subtle tooltip" subtle type="description">
+          <button>Subtle</button>
+        </Tooltip>
+        <Tooltip content="This tooltip has no arrow" noArrow type="description">
+          <button>No arrow</button>
+        </Tooltip>
+        <Tooltip
+          type="description"
+          content={
+            <>
+              This <i>tooltip</i> has <u>formatted</u> content
+            </>
+          }
+        >
+          <button>Formatted content</button>
+        </Tooltip>
+        <Tooltip content="This tooltip targets the red square" targetRef={targetRef1} type="description">
+          <button>
+            Custom target:{' '}
+            <div ref={targetRef1} style={{ display: 'inline-block', width: '8px', height: '8px', background: 'red' }} />
+          </button>
+        </Tooltip>
+        <Tooltip content="The trigger button was rendered by a render function" type="description">
+          {triggerProps => (
+            <div>
+              <button {...triggerProps}>Custom trigger</button>
+            </div>
+          )}
+        </Tooltip>
+      </div>
+    </FluentProvider>
+  );
+};
 
-const TooltipExampleCore = () => {
+export const Aria = () => {
   const styles = useStyles();
 
-  const refs = [
-    { tooltip: React.useRef<TooltipImperativeHandle>(null), target: React.useRef<HTMLDivElement>(null) },
-    { tooltip: React.useRef<TooltipImperativeHandle>(null), target: React.useRef<HTMLDivElement>(null) },
-    { tooltip: React.useRef<TooltipImperativeHandle>(null), target: React.useRef<HTMLDivElement>(null) },
-  ];
-
-  // eslint-disable-next-line no-restricted-properties
-  React.useLayoutEffect(() => {
-    refs.forEach(ref => {
-      if (ref.tooltip.current && ref.target.current) {
-        ref.tooltip.current.show(ref.target.current);
-      }
-    });
-  });
+  return (
+    <FluentProvider theme={webLightTheme}>
+      Use a screen reader to hear how the tooltip can be used as its target's label or description:
+      <div className={styles.exampleList}>
+        <Tooltip content="This tooltip is the label for the button" type="label">
+          {/* eslint-disable-next-line jsx-a11y/accessible-emoji */}
+          <button>💬</button>
```

Comment:
```suggestion
          <button><span role="img" aria-label="Comment">💬</span></button>
```

I suggest to avoid disabling this rule, we can just label this Emoji 😉

Reply:
This is meant to demo an icon button, where the tooltip is the label. I'll update it with an icon once we get the fluent icons hooked up. In the meantime I'll make the emoji be aria-hidden.